cfpb/consumerfinance.gov
# P4C: Tuition waiver field

Add a field for tuition waiver under grants & scholarships in grad path tool.


## Additions

- Field for tuition waiver in grants & scholarships sections
- URL param for tuition waiver field


## How to test this PR

1. Pull branch and run gulp
2. Navigate to [grad path tool](localhost:8000/paying-for-college/your-financial-path-to-graduation/?iped=110635&pid=4510-3&houp=onCampus&typp=bachelors&lenp=4&ratp=inState&depp=dependent&cobs=n&tuit=14184&hous=18222&book=849&houx=1395&fdx=498&clhx=127&trnx=750&hltx=265&entx=185&retx=282&taxx=143&othx=328)
3. Go to `Grants & scholarships` tab and verify that new `Tuition waiver` field appears under `Grants` section.
4. Add a value to `Tuition waiver` field and verify that it is subtracted from Uncovered costs
5. Reload page and verify that tuition waiver field still shows the value entered & value is still subtracted from uncoverd costs


## Screenshots


<img width="920" alt="e64c2280-71e9-11eb-9ff4-d62df15f9a5e" src="https://user-images.githubusercontent.com/778171/109228841-4eb17b80-7777-11eb-869a-c2560a268615.png">

## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
